### PR TITLE
docs: Mention Dockerfile based container prominently

### PR DESCRIPTION
**Pull request by okurz**
Related progress issue: https://progress.opensuse.org/issues/43718

### CODE REVIEW

File: docs/Installing.asciidoc [hunk 1]
```diff
@@ -84,6 +84,33 @@ zypper in openQA-bootstrap
 systemd-run -tM openqa1 /bin/bash # start a shell in the container
 -------------------------------------------------------------------------------
 
+== Container based setup
+[id="container_setup"]
+
+Containers are provided for both the webUI and worker.
```
Comment (Martchus) [inline docs/Installing.asciidoc:90]:
```suggestion
Containers are provided for both the web UI and worker.
```

We mainly use that spelling so for the sake of consistency. Besides, if you would write it out ("user interface") you would add a space as well so in my opinion it only makes sense to add a space.

File: docs/Installing.asciidoc [hunk 2]
```diff
@@ -84,6 +84,33 @@ zypper in openQA-bootstrap
...
+
+For example the webUI container can be pulled and started using the `podman`
```
Comment (Martchus) [inline docs/Installing.asciidoc:92]:
```suggestion
For example the web UI container can be pulled and started using the `podman`
```

File: docs/Installing.asciidoc [hunk 3]
```diff
@@ -84,6 +84,33 @@ zypper in openQA-bootstrap
...
+container engine:
+
+[source,sh]
+----
+podman run --rm -it registry.opensuse.org/devel/openqa/containers15.2/openqa_webui:latest
+----
+
+the worker container can be pulled and started with:
```
Comment (Martchus) [inline docs/Installing.asciidoc:100]:
```suggestion
The worker container can be pulled and started with:
```

File: docs/Installing.asciidoc [hunk 4]
```diff
@@ -84,6 +84,33 @@ zypper in openQA-bootstrap
...
+[source,sh]
+----
+podman run --rm -it registry.opensuse.org/devel/openqa/containers15.2/openqa_worker:latest
+----
+
+Some parameters are necessary to run them properly. See
+https://github.com/os-autoinst/openQA/blob/master/docker/README.md for more
+details and how to configure and run these containers.
+
+Take a look on
```
Comment (Martchus) [inline docs/Installing.asciidoc:111]:
```suggestion
Take a look at
```

**Review by Martchus — CHANGES_REQUESTED** [review]

File: docs/Installing.asciidoc (hunk 1 above)
Comment (okurz) [reply docs/Installing.asciidoc]: you are right. Thanks for your attention to detail.

**Comment by okurz** [comment]
fixed your suggestions

**Review by Martchus — APPROVED** [review]

**Review by ilausuch — APPROVED** [review]

**Pull request merged**